### PR TITLE
Run gofmt

### PR DESCRIPTION
**Pull request by flaper87**
This PR just fixes a fmt error

### CODE REVIEW

**Comment by flaper87** [comment]
We should add a job for gofmt and other lints/checks

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -29,5 +29,6 @@ aliases:
     - rjmorse
   cluster-api-openstack-maintainers:
     - dims
+    - flaper87
```
Comment (dims) [inline OWNERS_ALIASES]: Can we please move this to another PR?

Comment (flaper87) [reply OWNERS_ALIASES]: oh, that was accidental! Sorry :man_facepalming:

**Comment by flaper87** [comment]
Closing in favor of https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/38
